cfme/cloud/availability_zone.py

```diff
@@ -78,7 +78,7 @@ def is_displayed(self):
 
     toolbar = View.nested(AvailabilityZoneToolBar)
     entities = View.nested(AvailabilityZoneEntities)
-    paginator = View.nested(PaginationPane)
+    paginator = PaginationPane()
```

Comment:
ohh I see because of the changes to a VersionPick in PaginationPane. Gave myself -- because I implemented availability_zone and on my first pass through reading this I thought I had done it wrong :)